tests/errors/cpp_class_extern_overloaded_traits.pyx

```diff
@@ -0,0 +1,53 @@
+# mode: error
+# tag: cpp
+
+from libcpp cimport bool as bool_t
+
+cdef:
+	struct S1:
+		int i
+
+	struct S2:
+		int i
+
+	extern from "cpp_source.h":
```

Comment:
You do not need to add this file to `tests/errors/` since the C compiler will never be run after the translation fails in Cython.